cmd/version.go

```diff
@@ -38,23 +38,23 @@ func versionCommand() *cobra.Command {
 		Short: "Prints the version info",
 		Long:  "",
 		RunE: func(cmd *cobra.Command, args []string) error {
-
+			ctx := context.Background()
 			info := &Info{
 				Version: BuildVersionOverride,
 				Date:    BuildDate,
 				Commit:  BuildCommit,
 				License: "Apache-2.0",
 			}
 
-			// Where you are using go install, we will get good version information usefully from Go
+			// When we are using go install, we will get version information from Go
 			// When we're in go-releaser in a Github action, we will have the version passed in explicitly
 			if info.Version == "" {
 				buildInfo, ok := debug.ReadBuildInfo()
 				setBuildInfo(info, buildInfo, ok)
 			}
 
 			if shortened {
-				fmt.Println(info.Version)
+				log.L(ctx).Infof("Build info: %s", info.Version)
```

Comment:
I'm not sure that we'll get the version in this case, using CLI.
Will this info be printed in terminal, but not in the log file?